WordPress/Classes/Services/ReaderSiteService.h

```diff
@@ -28,25 +30,29 @@ extern NSString * const ReaderSiteServiceErrorDomain;
  Follow a wpcom site by ID.
 
  @param siteID The ID of the site.
- @param site Site that's being followed. Optional parameter.
+ @param topic Topic that's being followed. Optional parameter.
+ @param post Post that's being followed. Optional parameter.
```

Comment:
Is this a method to follow site, topics, or posts? I guess it's only sites but the `topic` and `post` params say otherwise.